.jenkins/pytorch/test.sh

```diff
@@ -306,6 +306,9 @@ test_inductor_benchmark_perf() {
     python benchmarks/dynamo/torchbench.py --performance --backend inductor --float16 --training \
       --batch-size-file "$(realpath benchmarks/dynamo/torchbench_models_list.txt)" --only hf_Bert \
       --output "$TEST_REPORTS_DIR"/inductor_training_$1.csv
+    # the reference speedup value is hardcoded in check_hf_bert_perf_csv.py
+    # this value needs to be actively maintained to make this check useful
+    python check_hf_bert_perf_csv.py -f inductor_training_$1.csv
```

Comment:
Do you need to add `"$TEST_REPORTS_DIR"` before the csv filename?

Reply:
Good catch, thank you! Yes, I will need to fix it.

Reply:
Done.